src/Umbraco.Core/Models/Relation.cs

```diff
@@ -17,12 +17,37 @@ public class Relation : EntityBase, IRelation
         private IRelationType _relationType;
         private string _comment;
 
+        /// <summary>
+        /// Constructor for constructing the entity to be created
+        /// </summary>
+        /// <param name="parentId"></param>
+        /// <param name="childId"></param>
+        /// <param name="parentObjectType"></param>
```

Comment:
I haven't used any GUID for relations because that is not how they are actually related in the DB. If we use GUID then you will need to lookup the INT as an extra DB call.

Reply:
But yes, the code comments are wrong - sorry VS 2019 hasn't got there yet to warn me since i have r# turned off